src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
+@import "./utils/variables";
+@import "./utils/functions";
+@import "./utils/mixins";
+
+@import "./blocks/header";
+@import "./blocks/cards";
+@import "./blocks/card";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
main font of page sets in html tag

Reply:
I think its can be done also in your way, but still all the content of the page is inside `body ` as well, so there not much difference, i suppose. 

Its only metters for property `font-size` if i or you will use rem or em, but i think they not very usfull, because the designer of the page not very often make some proportion for fonts)